Dockerfile

```diff
@@ -39,6 +39,11 @@ COPY docker/docker-entrypoint.sh docker-entrypoint.sh
 COPY startup_scripts/ /opt/netbox/startup_scripts/
 COPY initializers/ /opt/netbox/initializers/
 
+# Place default configuration in the container, so that it can be used
+# without docker-compose or older versions of docker-compose.yml
```

Comment:
Please remove the comments. IMO they don't tell us anything that we don't already know. (Any of the files can be replaces using a docker-compose mounting, so why bother writing this down?)

Reply:
(Then you can get rid of the extra blank link as well)